internal/service/anycast/model_proto_bgp_config.go

```diff
@@ -0,0 +1,127 @@
+package anycast
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/anycast"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type ProtoBgpConfigModel struct {
```

Comment:
Is this used? Please remove unused models?

Reply:
this will be used in anycast part 2